src/routes/tools/comparative-slr/_constants.js

```diff
@@ -0,0 +1,72 @@
+import { DEFAULT_BOUNDARIES } from "../_common/constants";
+
+export const DEFAULT_CENTER = [-122.2813, 37.7813];
+export const DEFAULT_FLOOD_SCENARIO = "med";
+export const DEFAULT_TIME_FRAME = "[2020,2040]";
+
+export const DEFAULT_INITIAL_CONFIG = {
+  boundaryId: "place",
+  floodScenario: DEFAULT_FLOOD_SCENARIO,
+  timeFrame: DEFAULT_TIME_FRAME,
+  lng: DEFAULT_CENTER[0],
+  lat: DEFAULT_CENTER[1],
+  imperial: false,
+};
+
+export const BOUNDARIES = DEFAULT_BOUNDARIES.filter((d) =>
+  ["counties", "censustracts", "hydrounits", "place"].includes(d.id)
+);
+
+export const TIME_PERIODS = [
+  {
+    id: "[2020,2040]",
+    label: "2020–2040",
+  },
+  {
+    id: "[2080,2100]",
+    label: "2080–2100",
+  },
+];
+
+export const FLOOD_SCENARIOS = [
+  {
+    id: "min",
+    label: "minimum",
+  },
+  {
+    id: "med",
+    label: "median",
+  },
+  {
+    id: "max",
+    label: "maximum",
+  },
+];
+
+export const DL_Cosmos = "cosmos";
```

Comment:
I like the idea of making these constants

Reply:
I find that using constants for strings (as well as magic numbers) that are used in multiple places helps reduce bugs. I was actually stuck on a bug for half a day this week because I hadn't realized I misspelled "visibility" in one place.